### ISSUE TITLE
Travis CI no longer recognizes `istanbul` command

### ISSUE THREAD

**Issue by cristiancavalli** [GoogleCloudPlatform/cloud-errors-nodejs#37]
It used to work, now it shows:
![screen shot 2016-07-22 at 2 55 03 am](https://cloud.githubusercontent.com/assets/4976075/17053644/d74135a4-4fb7-11e6-8c32-42b8eef4b807.png)

Try to fix by using the node_modules path:
`./node_modules/.bin/istanbul`

**Comment by cristiancavalli** [comment]
For some reason #36 wasn't pulled into the ci command
![screen shot 2016-07-22 at 3 19 32 am](https://cloud.githubusercontent.com/assets/4976075/17054302/43724f6c-4fbb-11e6-9ff1-d79bd3648c19.png)
Merging will hopefully cause it to use the right one

**Comment by cristiancavalli** [comment]
Update: travis ci seems to be pulling/caching old versions of the .travis.yml even after the job restart. I attempted to update my pull request .travis.yml to change this behavior but even though I repeated the git pull with success the old .travis.yml seems to be cached on travis ci.

My pull of the repo (.travis.yml)
![screen shot 2016-07-22 at 12 09 40 pm](https://cloud.githubusercontent.com/assets/4976075/17068376/836b7afe-5005-11e6-863d-e9004d4e5c7b.png)

What travis sees after the same git flow:
![screen shot 2016-07-22 at 12 12 29 pm](https://cloud.githubusercontent.com/assets/4976075/17068407/a25123e2-5005-11e6-860f-6fc7165d0dce.png)

Travis ci totally misses the new .travis.yml step.
Here are the git commands used (taken from the travis ci):

```bash
git clone --depth=50 https://github.com/GoogleCloudPlatform/cloud-errors-nodejs.git && \
cd cloud-errors-nodejs && \
git fetch origin +refs/pull/36/merge && \
git checkout -qf FETCH_HEAD
```
